.github/workflows/test.yml

```diff
@@ -74,3 +74,40 @@ jobs:
         env_vars: OS,PY,ISORT
       env:
         CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+  check:
+    name: tox env ${{ matrix.tox_env }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        tox_env:
+          - type
+          - dev
+          - docs
+          - pkg_meta
+          - readme
+        os:
+          - ubuntu-latest
+          - windows-latest
+        exclude:
+          - { os: windows-latest, tox_env: docs }
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Install the latest version of uv
+        uses: astral-sh/setup-uv@v5
+        with:
```

Comment:
Because we are using a lock file and by default the cache is enabled, you can remove the following three lines.